src/main/java/com/minecolonies/coremod/entity/ai/mobs/util/MobEventsUtils.java

```diff
@@ -107,6 +113,30 @@ else if (hordeTotal > MAX_SIZE)
         }
     }
 
+    private static BlockPos calculateFallBackSpawnLocation(final World world, final Colony colony)
+    {
+        if (colony == null)
+        {
+            Log.getLogger().debug("When trying to find SpawnPoint for Barbarian Event, the Colony was Null!");
+            return null;
+        }
+        final BlockPos center = colony.getCenter();
+        final int radius = Configurations.workingRangeTownHall;
+
+        final int x = center.getX() + radius;
+        final int y = center.getY();
+        final int z = center.getZ();
+
+        final BlockPos SpawnPoint = world.getTopSolidOrLiquidBlock(new BlockPos(x,y,z));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00117)